app/models/export/activity_forecast_columns.rb

```diff
@@ -48,9 +52,16 @@ def forecasts
   end
 
   def forecasts_to_hash
-    @_forecasts_to_hash ||= forecasts.each_with_object({}) { |forecast, hash|
-      hash[[forecast.parent_activity_id, forecast.financial_quarter, forecast.financial_year]] = forecast.value
-    }
+    @_forecasts_to_hash ||=
+      forecasts.each_with_object({}) { |forecast, hash|
+        hash[
+          [
+            forecast.parent_activity_id,
+            forecast.financial_quarter,
+            forecast.financial_year,
+          ]
+        ] = forecast.value
+      }
```

Comment:
I think this is just a formatting change now?

Reply:
yeah, trying to make it more readable!

Reply:
definitely an improvement in my book. 

The purist might say this formatting refactoring would be clearer in its own commit -- that way there's no effort expended in looking to understand what (if anything) has changed in this diff...